### DISCUSSION TITLE
incomplete output and break offs in the middle of a sentence

### DISCUSSION THREAD

**Discussion by nic0711** [danielmiessler/Fabric#841] [Q&A]
Hello!
I have noticed that my local ollama (llama3.1:8b-instruct-q8_0) installation only produces incomplete output and breaks off in the middle of a sentence.

For example: an output of “write_essay” produces a maximum of approx. 1200 characters. Or with “create_aphorisms” 12 instead of 20 aphorisms are output - the last one always incomplete.

I have already created a customized model with an extended context window (4096) - unfortunately without success.

Do you have any idea what I can do?

Thanks, Wolf

**Comment by xssdoctor** [comment]
Sometimes that happens to me and I just restart ollama

**Reply by nic0711** [reply]
Hi @xssdoctor 
thank you. I tried it - no difference

**Comment by nic0711** [comment]
I tested the concat mentioned here --> https://github.com/danielmiessler/fabric/discussions/825 - and it works fine... so my issue seems to have the same reason as described in this post --> https://github.com/danielmiessler/fabric/discussions/825
